spec/models/manageiq/providers/openstack/infra_manager/refresher_rhos_juno_spec.rb

```diff
@@ -5,11 +5,12 @@
 
   before(:each) do
     zone = EvmSpecHelper.local_miq_server.zone
-    @ems = FactoryBot.create(:ems_openstack_infra, :zone => zone, :hostname => "192.168.24.1",
-                              :ipaddress => "192.168.24.1", :port => 5000, :api_version => 'v2',
-                              :security_protocol => 'no-ssl')
+    credentials = Rails.application.secrets.openstack
+    @ems = FactoryBot.create(:ems_openstack_infra, :zone => zone, :hostname => credentials[:hostname],
+                              :ipaddress => credentials[:hostname], :port => credentials[:port].to_i, :api_version => 'v3',
```

Comment:
do we want to default the port?

```suggestion
                              :ipaddress => credentials[:hostname], :port => credentials[:port]&.to_i || 5000, :api_version => 'v3',
```

Reply:
I don't think so because `credentials` comes from `config/secrets.defaults.yml` and so will always be present.